<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>configureDialog</class>
 <widget class="QDialog" name="configureDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure FR24 Analyzer</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>481</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>11</y>
      <width>464</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>29</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>29</height>
        </size>
       </property>
       <property name="text">
        <string>From File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="textBrowser">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>29</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>301</width>
         <height>29</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="configureBrowseButton">
       <property name="minimumSize">
        <size>
         <width>75</width>
         <height>29</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>29</height>
        </size>
       </property>
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>341</width>
     <height>231</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>322</width>
      <height>204</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Geographical Paramters</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>29</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>29</height>
              </size>
             </property>
             <property name="text">
              <string>Airport Name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>29</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>29</height>
              </size>
             </property>
             <property name="text">
              <string>Airport Lattitude</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>29</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>29</height>
              </size>
             </property>
             <property name="text">
              <string>Airport Longtitude</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QTextEdit" name="airportName">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>29</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>29</height>
              </size>
             </property>
             <property name="toolTip">
              <string extracomment="Aiport Name Text (&quot;GUW&quot;)"/>
             </property>
             <property name="whatsThis">
              <string extracomment="Aiport Name Text (&quot;GUW&quot;)"/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="airportLat">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>29</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>29</height>
              </size>
             </property>
             <property name="toolTip">
              <string/>
             </property>
             <property name="whatsThis">
              <string extracomment="Format (xx.yyyyyy)"/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="airportLon">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>29</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>29</height>
              </size>
             </property>
             <property name="toolTip">
              <string/>
             </property>
             <property name="whatsThis">
              <string extracomment="Format (xx.yyyyyy)"/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>29</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>29</height>
            </size>
           </property>
           <property name="text">
            <string>Area</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="minimumSize">
              <size>
               <width>75</width>
               <height>29</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>75</width>
               <height>29</height>
              </size>
             </property>
             <property name="text">
              <string>Lattitude</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="minimumSize">
              <size>
               <width>75</width>
               <height>29</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>75</width>
               <height>29</height>
              </size>
             </property>
             <property name="text">
              <string>Longtitude</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="areaLayout">
         <item>
          <widget class="QTextEdit" name="areaLat">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>29</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>29</height>
            </size>
           </property>
           <property name="toolTip">
            <string/>
           </property>
           <property name="whatsThis">
            <string extracomment="Area Lattitude (Lat,Lon)"/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="areaLon">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>29</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>29</height>
            </size>
           </property>
           <property name="toolTip">
            <string/>
           </property>
           <property name="whatsThis">
            <string extracomment="Area Longtitude (Lat,Lon)"/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>341</width>
     <height>111</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>12</y>
      <width>146</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Network Paramters</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>35</y>
      <width>312</width>
      <height>68</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>29</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>29</height>
          </size>
         </property>
         <property name="text">
          <string>Redis IP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_13">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>29</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>29</height>
          </size>
         </property>
         <property name="text">
          <string>Postgres IP</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QTextEdit" name="redisIP">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>29</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>29</height>
          </size>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="whatsThis">
          <string extracomment="ip:port"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="postgresIP">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>29</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>29</height>
          </size>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="whatsThis">
          <string extracomment="ip:port"/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>421</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter the required paramteres or choose a config.yaml file.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveConfig">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>350</y>
     <width>100</width>
     <height>29</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>29</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>75</width>
     <height>29</height>
    </size>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
